packages/atomic/src/components/common/generated-answer/generated-answer-common.tsx

```diff
@@ -296,9 +306,7 @@ export class GeneratedAnswerCommon {
                     .answerStyle ?? 'default'
                 }
                 i18n={this.props.getBindings().i18n}
-                onChange={(answerStyle: GeneratedAnswerStyle) =>
-                  this.props.getGeneratedAnswer()?.rephrase({answerStyle})
-                }
+                onChange={this.onChangeAnswerStyle}
```

Comment:
I tried running the code locally, and clicking the rephrase buttons fail with this error:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'getGeneratedAnswerState')
```